Installation fails because of quotes around chdir

I'm getting following error when executing this role:

```
TASK: [ruby-install | Run ruby-install install target] ************************
failed: [default] => {
  "changed": true, 
  "cmd": ["make", "install", "chdir=/tmp/ruby-install-0.5.0"], 
  "delta": "0:00:00.009961", 
  "end": "2015-02-10 17:14:15.607230", 
  "rc": 2, 
  "start": "2015-02-10 17:14:15.597269", 
  "warnings": []
}
stderr: make: *** No rule to make target `install'.  Stop.
```

As you see `chdir...` is treated as part of the command so ansible doesn't change the current directory before executing `make install`.

ansible 1.8.2 on OSX 10.10.2
guest - Ubuntu 14.04.1 LTS

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -19,29 +19,28 @@
   # Install ruby-install
   #
 - name: Check for ruby-install runtime
-  stat: path="/usr/local/bin/ruby-install"
+  stat: path=/usr/local/bin/ruby-install
   register: ruby_install_runtime
 
 - name: Check for ruby-install tarball
-  stat: path="/tmp/ruby-install-{{ ruby_install_version }}.tar.gz"
+  stat: path=/tmp/ruby-install-{{ ruby_install_version }}.tar.gz
   when: not ruby_install_runtime.stat.exists
   register: ruby_install_tarball
 
 - name: Download ruby-install
-  get_url: url="http://github.com/postmodern/ruby-install/archive/v{{ ruby_install_version }}.tar.gz"
-           dest="/tmp/ruby-install-{{ ruby_install_version }}.tar.gz"
+  get_url: url=http://github.com/postmodern/ruby-install/archive/v{{ ruby_install_version }}.tar.gz
+           dest=/tmp/ruby-install-{{ ruby_install_version }}.tar.gz
   when: not ruby_install_runtime.stat.exists and not ruby_install_tarball.stat.exists
   register: ruby_install_downloaded
 
 - name: Untar ruby-install
-  command: tar xf "/tmp/ruby-install-{{ ruby_install_version }}.tar.gz"
-           chdir="/tmp"
+  command: tar xf /tmp/ruby-install-{{ ruby_install_version }}.tar.gz
+           chdir=/tmp
   when: ruby_install_downloaded | changed
   register: ruby_install_untarred
 
 - name: Run ruby-install install target
   sudo: yes
   command: make install
-           "chdir=/tmp/ruby-install-{{ ruby_install_version }}"
+           chdir=/tmp/ruby-install-{{ ruby_install_version }}
   when: ruby_install_untarred | changed
-

